src/actions/publish.js

```diff
@@ -95,6 +95,11 @@ async function storeJustPublishedProfileData(
   sanitizedInformation,
   prepublishedState: State
 ): Promise<void> {
+  if (process.env.NODE_ENV === 'test' && !window.indexedDB) {
```

Comment:
Thought: `storeJustPublishedProfileData` never mentions in the function name nor in the comments *where* the profile is stored (e.g. locally in a IndexedDB), so it lacks a bit of context when coming across the function in a code review.

This is also the case with the function `storeProfileData` below.

Reply:
Yeah that's a good feedback. I'll look at it (probably in a separate PR though).